templates/letsencrypt.timer.erb

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
please add a comment at the top like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
Done.